configure

```diff
@@ -671,6 +671,15 @@ def configure_winsdk(o):
   print('ctrpp not found in WinSDK path--using pre-gen files '
         'from tools/msvs/genfiles.')
 
+
+def configure_wix(o):
+  if flavor != 'win':
+    return
+
+  o['variables']['wix_bin_dir'] = os.path.abspath(os.path.join(
+      os.environ.get('WIX'), 'bin'));
```

Comment:
If `WIX` is not there in the dictionary we will get `NoneType doesn't have ...` error. It's better to fail with `KeyError` in that case.